go.mod

```diff
@@ -1,40 +1,41 @@
 module kubevirt.io/ssp-operator
 
 go 1.21
+toolchain go1.22.2
```

Comment:
Before we merge this, we should bump the golang version. @akrejcir WDYT?

Reply:
Yes, or we should disable dependabot, because it is failing every day.
There are already downstream base images with golang 1.22, so we can update upstream.